axelor-base/src/main/java/com/axelor/apps/base/web/ImportCityController.java

```diff
@@ -18,51 +18,70 @@
 package com.axelor.apps.base.web;
 
 import com.axelor.apps.base.db.ImportHistory;
+import com.axelor.apps.base.exceptions.IExceptionMessage;
 import com.axelor.apps.base.service.imports.ImportCityService;
-import com.axelor.exception.service.TraceBackService;
-import com.axelor.inject.Beans;
-import com.axelor.meta.MetaFiles;
-import com.axelor.meta.db.MetaFile;
-import com.axelor.meta.db.repo.MetaFileRepository;
+import com.axelor.exception.AxelorException;
+import com.axelor.i18n.I18n;
+import com.axelor.meta.schema.actions.ActionView;
 import com.axelor.rpc.ActionRequest;
 import com.axelor.rpc.ActionResponse;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
-import java.io.File;
-import java.nio.charset.StandardCharsets;
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
 import java.util.LinkedHashMap;
-import org.apache.commons.io.FileUtils;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Singleton
 public class ImportCityController {
 
   @Inject private ImportCityService importCityService;
 
+  // Logger log = Logger.getLogger(ImportCityController.class);
+
+  private final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
log is not getting used anywhere in the controller.
if it is not required remove it.